tests/presets/test_datapackages.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from goodtables import presets
+
+
+# Test
+
+def test_datapackages():
+    errors, tables = presets.datapackages([
```

Comment:
I've reverted it - https://github.com/frictionlessdata/goodtables-py/commit/1d35ef736afcad125b2d6a815a47b20db28a50e0 - analyzed our `validation_conf` schema and decided be more strict for now.